bundles/org.smarthomej.binding.telenot/src/main/java/org/smarthomej/binding/telenot/internal/handler/TelenotBridgeHandler.java

```diff
@@ -303,6 +304,10 @@ private void readerThread() {
                 } catch (MessageParseException e) {
                     logger.warn("Error {} while parsing message {}. Please report bug.", e.getMessage(), message);
                 }
+                if (discoveryStarted && usedInputContact.isEmpty() && usedReportingArea.isEmpty()) {
+                    discoveryStarted = false;
+                    logger.info("Discovery job completed");
```

Comment:
Reduce to debug

Reply:
I want to show the user that the discovery is in progress. It takes about 5 minutes to discover all channels, so it would be nice to know when it will be completed.